addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComHumidityMessage.java

```diff
@@ -154,64 +143,37 @@ public String getDeviceId() {
     }
```

Comment:
What does Janne have to do with decoding signal and battery level at line 135?

```
        data[8] = (byte) (((signalLevel & 0x0F) << 4) | (batteryLevel & 0x0F));// Janne
```

Reply:
@paulianttila do you have any memory of this:

See:

https://github.com/openhab/openhab1-addons/blob/471e98e578fbc6dddba2886f2551b789b18d03ad/bundles/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComHumidityMessage.java#L158

Reply:
Maybe Janne was an author? https://github.com/openhab/openhab1-addons/commit/8324901c7ae18a72bd9278f66dfc031cdf111c12#diff-43da6f3b42811ea145e4117bcb6ceaffR14

Reply:
Yes there it is 

https://groups.google.com/d/msg/openhab/zcWacaQnE_c/qv-2mvKi2rQJ